src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -63,9 +69,14 @@ export default function OpeningPeriodForm({
       ? formatApiTimeSpansToFormFormat(firstTimeSpanGroup.time_spans)
       : [{}];
 
+  const rules: GroupRule[] | [] = firstTimeSpanGroup?.rules.length
```

Comment:
While reading this I was confused why rules come only from the first time span group, but then I realised we only have one time span group at moment.

Reply:
Yes, we still use only one group 😁